includes/class-payment-information.php

```diff
@@ -144,15 +144,15 @@ class Payment_Information {
 	 */
 	public function __construct(
 		string $payment_method,
-		\WC_Order $order = null,
-		Payment_Type $payment_type = null,
-		\WC_Payment_Token $token = null,
-		Payment_Initiated_By $payment_initiated_by = null,
-		Payment_Capture_Type $manual_capture = null,
-		string $cvc_confirmation = null,
+		?\WC_Order $order = null,
+		?Payment_Type $payment_type = null,
+		?\WC_Payment_Token $token = null,
+		?Payment_Initiated_By $payment_initiated_by = null,
+		?Payment_Capture_Type $manual_capture = null,
+		?string $cvc_confirmation = null,
 		string $fingerprint = '',
```

Comment:
@marcinbot Would this cause an issue since it's not optional or nullable, but does have a default value and sits between optional params.